about.html

```diff
@@ -0,0 +1,75 @@
+</head>
+
+<body>
+    <header>
+    </header>
+    <main class="container`-fluid bg-gray row mx-0">
+        <section class="vh100 align-items-center justify-content-center">
+            <h1>ABOUT US</h1>
+            <p class="bg-white py-3 px-3 border-gray my-3 col-md-10 col-lg-8">Our name was chosen to convey the
+                restaurant’s
+                welcoming
+                environment,
+                as it truly
+                is
+                «the
+                address» for
+                friends, colleagues and tourists to meet up at any time of day. At BlaBla we serve innovative breakfast,
+                brunch, lunch and dinner
+                menus, as well as pâtisserie created on site in the classic French tradition. Our diverse beverage
+                program
+                includes artisanal cocktails, fruit and vegetable blends, spritzers and one-of- a-kind coffee drinks
+                prepared by an award-winning barista team.</p>
+            <p>Please contact us by Email for any questions and suggestions.<br><span
+                    class="fw-bold">blabla@restaurant.com</span></p>
+        </section>
+        <section class="align-items-center justify-content-start">
+            <div class="h2-wrapper">
+                <h2>Logo</h2>
+                <div class="line"></div>
+            </div>
+            <p class="col-md-10 col-lg-8">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Soluta non veniam
+                culpa quibusdam eius laboriosam amet placeat ipsam quia similique autem perferendis, aliquid nobis
+                deserunt fugiat eligendi! Officia, magnam dolorem.</p>
+            <div class="border border-1 my-4 col-10 col-sm-7 col-md-5 col-lg-3">
+                <img class="img-fluid" src="./images/logo-food.png" alt="">
+            </div>
+        </section>
+        <section class=" container gap-4" id="team">
+            <div class="h2-wrapper">
+                <h2>Our Team</h2>
+                <div class="line"></div>
+            </div>
+            <div class="gy-4 gx-4 row row-cols-1 row-cols-md-2" id="team-members"></div>
+        </section>
+        <section id="partners"
+            class="mt-0 gy-4 row row-cols-3 row-cols-sm-4 flex-row justify-content-center align-items-center">
+            <div class="h2-wrapper my-0">
+                <h2 class="text-white">Partners</h2>
+                <div class="line mb-0"></div>
+            </div>
+            <img class="col" src="./images/logo/gray/580b57fcd9996e24bc43c4f2-Copy.png" alt="">
+            <img class="col" src="./images/logo/gray/587b518e44060909aa603a8c-Copy.png" alt="">
+            <img class="col" src="./images/logo/gray/589a41155aa6293a4aac48a8-Copy.png" alt="">
+            <img class="col" src="./images/logo/gray/PikPng.com_amazon-logo-png_1037301-Copy.png" alt="">
+        </section>
+    </main>
+    <footer class="copyright py-4 px-3 text-start d-flex justify-content-center">
```

Comment:
- [ ] While adding the `footer` section you mismatch the background of desktop and mobile versions. Kindly fix this. 🔧 
- **Requirement**

![image](https://user-images.githubusercontent.com/26767383/152220093-bea7efa2-07e9-4c2b-9680-a8608166ddc8.png)

Reply:
Ok fixed this